pom.xml

```diff
@@ -41,8 +41,6 @@
     </scm>
     <properties>
         <?SORTPOM IGNORE?>
-        <java.target.version>8</java.target.version>
-        <jdk.min.version>11</jdk.min.version>
```

Comment:
why not set them both to 11? Is it defined somewhere else?

Reply:
yes. in the parent pom https://github.com/eclipse/che-parent/blob/master/pom.xml#L102-L103